addon/services/objectlistview-events.js

```diff
@@ -24,6 +24,15 @@ export default Ember.Service.extend(Ember.Evented, {
   */
   _selectedRecords: undefined,
 
+  /**
+    Table columns related to current model projection.
+
+    @property columns
+    @type Object[]
+    @private
+  */
+  _columns: undefined,
```

Comment:
The service is one for all, you need to add separation by components.
Use the object and component names as its properties.